airflow/models/dag.py

```diff
@@ -3204,6 +3204,13 @@ def __init__(self, concurrency=None, **kwargs):
     def __repr__(self):
         return f"<DAG: {self.dag_id}>"
 
+    @property
+    def schedule_description(self) -> str | None:
```

Comment:
This is not enough, dataset triggered dag also have a schedule_interval as a string, I think we need to check the self.timetable attribute.
![image](https://user-images.githubusercontent.com/14861206/206874517-4ec14250-3da2-492d-ac80-d86cd0582abd.png)

Reply:
@pierrejeambrun the DagModel class doesn't have self.timetable attribute. I used self.timetable_description instead.